Content.Server/Forensics/Components/DnaComponent.cs

```diff
@@ -6,6 +6,6 @@
 [RegisterComponent]
 public sealed class DnaComponent : Component
 {
-    [DataField("dna")]
-    public string? DNA = String.Empty;
+    [ViewVariables(VVAccess.ReadWrite)]
```

Comment:
i meant add it not replace sorry
do `[DataField("dna"), ViewVariables(VVAccess.ReadWrite)]`

Reply:
oh, misunderstood u, sorry, fixed now